### DISCUSSION TITLE
Integrate this with ChatGPT

### DISCUSSION THREAD

**Discussion by FlamxGames** [FlamxGames/godot-ai-assistant-hub#2] [Ideas]
Integrating this with ChatGPT could be super easy, you pretty much need to create a new file implementing `LLMInterface` class. You can check class `OllamaAPI` as an example. I made a video explaining the few things you would need to do, so check it out if you are interested.

Just try to write clean code and be open to feedback.

**Comment by kangoomotto** [comment]
I will definitely give it a go. Where can I share the implementation?

**Reply by FlamxGames** [reply]
You can do a pull request here in GitHub. You can check a previous PR for reference [here](https://github.com/FlamxGames/godot-ai-assistant-hub/pull/12).
